frontend/common/src/constants/localizedConstants.tsx

```diff
@@ -45,6 +45,12 @@ export const disabilityLocalized = {
   description: "Message for person with a disability",
 };
 
+export const requiredFieldsMissing = defineMessage({
```

Comment:
Looking at this again, might `requiredFieldsMissing` be better suited for [commonMessages.ts](https://github.com/GCTC-NTGC/gc-digital-talent/blob/main/frontend/common/src/messages/commonMessages.ts)?

Reply:
I'll move it there. In the meantime, can you explain the difference between the two files? Like when would you put something in `localizedConstants.tsx` over `commonMessages.ts`?

Reply:
from the wisened @petertgiles:
> localized constants is mainly for enums from the graphql schema.  Since they don't live in the database we can't store an {en: '', fr: ''} for them there.

Reply:
Thanks!

Moved with [move localized message to common messages](https://github.com/GCTC-NTGC/gc-digital-talent/pull/3459/commits/aef9a40a93a7358a7f845b43d68c2abd797ac3de)